feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/balance/detail/BalanceDetailFragment.kt

```diff
@@ -105,8 +105,6 @@ class BalanceDetailFragment : BaseFragment<BalanceDetailViewModel>() {
     }
 
     override fun subscribe(viewModel: BalanceDetailViewModel) {
-        viewModel.sync()
```

Comment:
Why was it causing the mentioned bug?

Reply:
Fragment calls onViewCreated after back navigation so subscribe is called every time we go back from transaction details